.gitignore

```diff
@@ -82,3 +82,8 @@ tags
 .kube/*
 /build
 .qz/*
+
+### module helm chats ###
+boot/helm-charts/slimeboot/templates/lazyload/*
```

Comment:
maybe we can use `boot/helm-charts/slimeboot/templates/modules/` and put module subcharts under that.